src/rep.c

```diff
@@ -1,11 +1,17 @@
+#include "rlang.h"
 #include "vctrs.h"
 #include "type-data-frame.h"
 #include "decl/rep-decl.h"
+#include "vec.h"
```

Comment:
```suggestion
#include "vec.h"
#include "decl/rep-decl.h"
```

I think we always want decl/ headers last?